pkg/cmd/create/create_vault.go

```diff
@@ -252,6 +253,49 @@ func (o *CreateVaultOptions) CreateVault(vaultOperatorClient versioned.Interface
 	return nil
 }
 
+func (o *CreateVaultOptions) setCloudProviderSpecificSettings(vaultCRD *v1alpha1.Vault, kubeProvider string, clusterName string, ns string, vaultAuthServiceAccount string, kubeClient kubernetes.Interface, vaultOperatorClient versioned.Interface) error {
+	var cloudProviderConfig kubevault.CloudProviderConfig
+	var err error
+
+	if o.IsJXBoot() {
+		// only when executing locally via `jx boot` we expect to have the appropriate credentials to modify cloud resources
+		// the pipeline user won't have permissions to execute cloud provider apis
+		switch kubeProvider {
+		case cloud.GKE:
+			cloudProviderConfig, err = o.vaultGKEConfig(vaultCRD, vaultOperatorClient, o.BucketName, o.KeyringName, o.KeyName, kubeClient, clusterName, vaultAuthServiceAccount)
+		case cloud.AWS, cloud.EKS:
+			cloudProviderConfig, err = o.vaultAWSConfig(vaultCRD, vaultOperatorClient, kubeClient, clusterName, vaultAuthServiceAccount)
+		default:
+			err = errors.Errorf("Vault is not supported on cloud provider %s", kubeProvider)
```

Comment:
error message is no capitalised in Go

Reply:
I know, but is this not Vault as Vault the product? But I can change it